### DISCUSSION TITLE
how to get the doris version info

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [apache/doris#44531] [A - General / Q&A] [answered]
The doris depolyed by another team, and now I want to know the version info .I am using this command to get the version info:

`select version();`

and output like this: `5.7.99`, I read the official document and found the version info from here: https://doris.apache.org/community/release-versioning

how to get the front end and backend version info or full version info of doris? the newest version is 3.x.x why the version is `5.7.99`?

**Comment by Vallishp** [comment]
show frontends;  // will have fe level version /commit hash
show backends; // will have be level version/commit hash

**Comment by echo-hhj** [comment, marked as answer]
1. `select @@version_comment;`
2. `show frontends;`
3. `show backends;`
4. `./start_fe.sh --version` (The commit information will be printed in the console or `fe.out`.)

**Comment by melin** [comment]
参考 starrocks 提供 current_version 函数。返回 doris 版本

**Comment by feiniu55666** [comment]
对呀，这个5.7.99 是个什么？

**Reply by jiangtao69039** [reply]
是 mysql 的东西吧

**Reply by MrZhaoooo** [reply]
the driver's version,probably
